Libraries/Text/Text.js

```diff
@@ -177,6 +177,10 @@ const Text = React.createClass({
      */
     testID: PropTypes.string,
     /**
+     * If true, plays system sound on touch
```

Comment:
Because `Text` has `onPress` too, and it can by used without being wrapped in touchable. If something can handle press, then it should be able to make sound. Besides, in discussion of first PR https://github.com/facebook/react-native/pull/6825 `Text` was used in internal tests.